### PR TITLE
Add Roundtable Call Guidelines

### PR DESCRIPTION
**Pull request by dcwalk**
This addresses https://github.com/datatogether/roadmap/issues/76

### CODE REVIEW

File: guidelines/round-table.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Data Together Weekly Roundtable Call
+
+Once a week on **Mondays 5-5:30pm**, a rotating facilitator will lead a round table discussion of a Data Together-related topic, challenge, or question. We use a standing [Zoom call link](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201): [zoom.us/j/455833201](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201) and [GitHub `roundtable` issues](https://github.com/datatogether/datatogether/labels/roundtable) to run each call. **Check out [how to host](#how-to-host)** before running your first call.
+
+### Objectives
```
Comment (patcon) [inline guidelines/round-table.md]: Your call, but maybe bolding operative bits of each bullet? ie. "ongoing work", "larger issues", "developing ideas", "maintain momentum"? You have made terse-skills dc, but thinking about the skimmers :)

File: guidelines/round-table.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- Check-in on ongoing work and potentially surface larger issues that need to be discussed in-depth
+- Allow initial feedback on developing ideas prior to opening a decision
+- Maintain momentum going between larger coordinating calls
+
+### Format
+
+Meetings will rotate individuals leading a round table around a specific topic, problem, or task. Scheduling topics will interleave implementation, governance, and topic area questions. Round table topics will be picked at least a week in advance.
```
Comment (patcon) [inline guidelines/round-table.md]: not totally clear whether rotation is in topics or individuals or both

Comment (patcon) [reply guidelines/round-table.md]: > Scheduling topics will interleave implementation, governance, and topic area questions.

I'm sorry if I'm being dense, but I don't know what this sentence means!

File: guidelines/round-table.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Example Round Table Topics:
+- checking in on decision-making process
+- reviewing website copy
+- asking about and open PR or design detail related to a reference implementation
+- identifying partnership & outreach opportunities
+
+#### Timing
+
+- **Mondays 5-5:30pm**
+- Secondary slot: Thursdays 5-5:30pm
```
Comment (patcon) [inline guidelines/round-table.md]: When do we use the secondary slot? Or maybe that doesn't matter! (I'm thinking it's data from the when2meet that you want to capture somewhere more permanent :slightly_smiling_face: )

File: guidelines/round-table.md (hunk 2 above)
Comment (patcon) [reply guidelines/round-table.md]: Might this work? Sorry, hard to know if it's any better, but my mind stumbled on "rotate individuals leading a round table around" :)

> Meetings will rotate leaders for each round table on a specific topic, problem, or task.

File: guidelines/round-table.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+- Intros and bullet-point updates (10mins)
+- Round Table (15mins)
+  - aim for new leader each week
+- Wrap-up (5mins)
+  - identify TODOs
+  - review upcoming meeting schedule
+
+### How to Host
+
+1. Start a GitHub issue in [`datatogether/datatogether`](https://github.com/datatogether/datatogether/issues/new) with the title "Roundtable: [Topic] [Date (optional)]"
```
Comment (patcon) [inline guidelines/round-table.md]: "Roundtable" or "round table"?

I like roundtable more, fwiw!

**Comment by patcon** [comment]
This looks awesome dawn! I read this with a tired brain, so def use your own judgement for whether any of my comments merit change :)

File: guidelines/round-table.md (hunk 2 above)
Comment (dcwalk) [reply guidelines/round-table.md]: thanks, yeah this was messy. Hopefully better!

File: guidelines/round-table.md (hunk 3 above)
Comment (dcwalk) [reply guidelines/round-table.md]: This was just data from when2meet. If Mondays ever become an issue :)

File: guidelines/round-table.md (hunk 4 above)
Comment (dcwalk) [reply guidelines/round-table.md]: roundtable it is :)

**Comment by dcwalk** [comment]
Okay, addressed @patcon's feedback, will tackle adding something to readme separately, as I think we want a wee overhaul of how info is framed here (but see that as less urgent).

Going to merge so we have for our call tonight!

**Pull request merged**